### DISCUSSION TITLE
Will autocomplete be added to the console enabler?

### DISCUSSION THREAD

**Discussion by acyc070** [UE4SS-RE/RE-UE4SS#515] [Ideas]
![console](https://github.com/UE4SS-RE/RE-UE4SS/assets/112773735/4e8a9e87-b5fb-4904-a6e8-9fda34f16eb2)

**Comment by UE4SS** [comment]
I don't know how the console auto-complete system works in UE, nor why it doesn't work when we re-enable the console.
It's a low priority thing so if we decide to look into this, it likely won't happen for some time.

**Comment by Buckminsterfullerene02** [comment]
UE4SS only re-enables the in-game console by bridging the gap created in shipping games - typically, the console is shipped with the game, but with just no way to enable it. 

It is definately out of scope of UE4SS to edit the functionality of the console, and I can imagine it would take considerable work to edit/hook engine functions to somehow re-enable this functionality or whatever else depending on how it's been stripped from the game. Definately does not sound worth the development effort, unless one of the nutters in the modding scene sees it as a challenge and decides to work on it anyway, in which case, you'll be in luck 🙂
